psydac/linalg/basic.py

```diff
@@ -66,10 +66,35 @@ def zeros(self):
 
         """
 
-#    @abstractmethod
-    def dot(self, a, b):
+    @abstractmethod
+    def inner(self, x, y):
         """
-        Evaluate the scalar product between two vectors of the same space.
+        Evaluate the inner vector product between two vectors of this space V.
```

Comment:
'this' space: which one ?

Reply:
The one mentioned in the other docstrings? This is a method of the `VectorSpace` class, so there is just one space.